compiler/luci-interpreter/src/kernels/Add.test.cpp

```diff
@@ -40,6 +40,8 @@ TEST(AddTest, Float)
        1.0f, 0.0f, 1.1f, 0.0f, 0.4f, 0.0f, 1.7f, 3.3f, 2.2f, 3.8f, 2.1f, 3.7f,
        0.0f, 0.5f, 0.0f, 1.0f, 0.0f, 0.9f, 1.2f, 0.0f, 1.7f, 0.0f, 1.6f, 0.0f},
       {0.0f, 2.5f, 1.2f, 0.8f, 0.4f, 0.0f, 1.7f, 3.3f, 0.0f, 1.0f, 1.6f, 0.0f}};
+  std::vector<std::vector<int>> test_shape_outputs = {
```

Comment:
I would suggest `expected_output_shapes` or `reference_output_shapes` or shortened versions `exp_output_shapes` and `ref_output_shapes`.

Reply:
I think ref_output_shape would be better.